incubator/java-microprofile/image/project/pom.xml

```diff
@@ -62,12 +62,12 @@
                     <configuration>
                         <rules>
                             <requireJavaVersion>
-                                <version>[1.8.0-212]</version>
+                                <version>(1.8,1.9)</version>
```

Comment:
So the parenthesis () means the java version must be between _not including_ 1.8 and 1.9
Do we want to include the 1.8 with a square bracket [ instead?
`[1.8,1.9)`